packages/rulesets/package.json

```
@@ -86,4 +86,4 @@
       "^@stoplight/spectral-ruleset-bundler/(.*)$": "<rootDir>/node_modules/@stoplight/spectral-ruleset-bundler/dist/$1"
     }
   }
-}
+}
```

```suggestion
}

```

Add trailing newline